daemons/dss-sync/app.py

```diff
@@ -1,11 +1,17 @@
 import os, sys, json, boto3, domovoi
 
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), 'domovoilib'))
+sys.path.insert(0, pkg_root)
+
+from dss.events.handlers import sync_blob # noqa
+
 app = domovoi.Domovoi()
 
 s3_bucket = os.environ.get("DSS_S3_TEST_BUCKET")
 
 @app.s3_event_handler(bucket=s3_bucket, events=["s3:ObjectCreated:*"])
 def process_new_syncable_object(event, context):
-    bucket = boto3.resource("s3").Bucket(event['Records'][0]["s3"]["bucket"]["name"])
+    s3 = boto3.resource("s3")
+    bucket = s3.Bucket(event['Records'][0]["s3"]["bucket"]["name"])
```

Comment:
The docs at http://docs.aws.amazon.com/AmazonS3/latest/dev/notification-content-structure.html don't explicitly state it, but from what I know we can expect to receive no more than one record at a time.